src/viewer.js

```diff
@@ -437,65 +732,77 @@ class VolumeImageViewer {
 
     // Collection of Openlayers "Feature" instances
     this[_features] = new Collection([], { unique: true });
+
     // Add unique identifier to each created "Feature" instance
-    this[_features].on('add', (e) => {
+    this[_features].on("add", (e) => {
       // The ID may have already been set when drawn. However, features could
       // have also been added without a draw event.
       if (e.element.getId() === undefined) {
         e.element.setId(generateUID());
       }
+
+      this[_annotationManager].onAdd(e.element);
+    });
+
+    this[_features].on("remove", (e) => {
+      this[_annotationManager].onRemove(feature);
```

Comment:
The variable `feature` is not defined. I think it should be:

```suggestion
      this[_annotationManager].onRemove(e.element);
```

Reply:
Its fixed on my local env, ill push soon, thanks!